### ISSUE TITLE
News on consumer site - opens page with Broker nav

### ISSUE THREAD

**Issue by stephnyc** [zoodigital/nia#52]
Clicked on the latest news item on the homepage (consumer) http://newireland.2ds.in/news/increased-initial-commission-promise-plus-10/

It opens with broker dashboard nav above it. Note: I dont have broker homepage as default or anything.

**Comment by seonzoo** [comment]
The news links will open with the broker template if the cookie 'isBroker' is set

This is true if you accept the broker popup and has no relation to the other cookie which sets broker homepage.

**Comment by stephnyc** [comment]
I'm not sure that is the correct UX - anyone could click yes on the popup and we really dont want them being presented with the broker nav 

Could it be based on the category of the news item? If if broker only selected then broker nav else (i.e: 
consumer or consumer+broker) then consumer nav? Lets discuss with MB

**Comment by stephnyc** [comment]
Sprint #2 as UX TBC

**Comment by seonzoo** [comment]
if anyone clicks yes on the broker page they will see the broker nav and full dashboard there too!

**Comment by stephnyc** [comment]
yes they will! but they may just have been curious or something.. no reason to expect them to be a broker and give them broker navigation at a later point

can you confirm, how long is "isBroker" set? each session or once? or what? thanks
